subsys/net/lib/lwm2m/lwm2m_rw_senml_json.c

```diff
@@ -357,87 +379,40 @@ static int put_char(struct lwm2m_output_context *out, char c)
 	return 1;
 }
 
-static int put_json_prefix(struct lwm2m_output_context *out, struct lwm2m_obj_path *path,
+static int put_senml_name_type(struct lwm2m_output_context *out, struct lwm2m_obj_path *path,
```

Comment:
I don't understand the reasoning behind the changes of the naming conventions.
`put_json_prefix` -> `put_senml_name_type`
`put_json_postfix` -> `put_json_end_object`
could you shed some light on those?

Reply:
OK Let me re-think those.  postfix is adding end of object and prefix is adding object start, basename, name, type
Midle of those is payload write